microsoft/testsuites/dpdk/dpdktestpmd.py

```diff
@@ -101,6 +102,9 @@ def can_install(self) -> bool:
     def dependencies(self) -> List[Type[Tool]]:
         return [Git, Wget]
 
+    def set_dpdk_source(self, dpdk_source: str) -> None:
+        self._dpdk_source = dpdk_source
```

Comment:
Not to use class level variable to save this information. It may break when run in parallel.

Reply:
I see, I thought I couldn't add parameters to install since it was a parent function but I CAN pass in additional kwargs. I'll change these to function arguments for install()